MIGRATION_GUIDE.md

```diff
@@ -3,6 +3,35 @@
 
 This guide outlines how developers can migrate from older versions of our SDK to newer ones.
 
+## Migrating to v3.0.0
```

Comment:
Is this section needed if it just references migrating to v2.4.0 anyway?

Reply:
Oh actually I thought this was in the setup readme, I see you actually put it in a standalone migration guide. I think thats more sensible!

Reply:
Evan added the migration guide :) 
This PR was suppose to just be just updating SDK version in code to 3.0.0 and because master was ahead of the release branch there was some file that got included here like the one you commented on.